g2p/apply.py

```diff
@@ -56,20 +61,61 @@ def __init__(
         self.variants_number = variants_number
         self.word_list = word_list_file
         self.filter_empty_words = filter_empty_words
+        self.concurrent = concurrent
+
+        self.out_g2p_lexicon_parts = None
```

Comment:
The parts can be intermediate files just stored in `work` , there is no need to have this as output. Or do you want to use those explictely in your pipeline later on?

Reply:
No, they don't need to be outputs. Moved them to work now.